src/components/CategoryFilter.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+
+const CategoryFilter = props => {
+  const categories = ['All', 'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  const { handleFilter } = props;
+
+  const handleClick = event => {
+    const { value } = event.target;
+    // handleFilter(value);
+    // console.log(value);
```

Comment:
**REQUIRED**
Comments: It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from Pathwright)